README.md

```diff
@@ -1 +1,14 @@
+# Testing
 
+Testing can be run either through the launch script, or manually.
+To use the launch script,run the following command in the root folder:
+
+```bash
+. scripts/launch_frontend.sh
+```
+
+To launch the frontend manually, run the following command in rover-Basestation-Frontend:
+
+```bash
+npm run dev -- --open
```

Comment:
we want to build the files and ONLY build them. the built files should be placed in our output directory (again, see my main comment for this review)